tools/system_calls.py

```diff
@@ -139,3 +163,38 @@ def get_screen():
     buffered_screen = BytesIO()
     screen.save(buffered_screen, format='JPEG')
     return base64.b64encode(buffered_screen.getvalue()).decode('utf-8')
+
+
+def url_history(url):
+    """Stores 10 last casted url
+
+    :param url: URL to save
+    :type url: str
+
+    :return: List of urls
+    :rtype: list
+    """
+    if url is '' or None or []:
+        pass
```

Comment:
It’ll raise error on return because no *urls* variable will be defined

Reply:
Sorry, this is defined at the top of script.
It’s really confusing.

You can for example create file with urls that will be persistent even on restart of teleserver.